src/Common/tests/System/Net/Prerequisites/Servers/CoreFxNetCloudService/WebServer/WebSocket/EchoWebSocket.ashx.cs

```diff
@@ -15,12 +15,15 @@ namespace WebServer
     public class EchoWebSocket : IHttpHandler
     {
         private const int MaxBufferSize = 128 * 1024;
+        private bool _replyWithPartialMessages;
```

Comment:
>Should I initialize it inline or add an explicit constructor and initialize it there?

When we initialize fields in a class, we usually do it inline in the class definition.method, similar to how the `const int MaxBufferSize` was done.